src/app/archive_blocks/dune

```diff
@@ -3,12 +3,27 @@
  (name archive_blocks)
  (public_name archive_blocks)
  (libraries
+   ;opam
```

Comment:
Nit: these comments would be clearer formatted like `;; opam`, to distinguish them from 'todo' commented-out stanzas (which I suspect we have several of).

Reply:
Thanks, I fixed these and merged `develop` in to keep things up to date.